projects/kit/components/rating/rating.template.html

```diff
@@ -0,0 +1,33 @@
+<div class="t-rating-interactive">
+    <tui-svg
+        *tuiRepeatTimes="let index of max"
+        class="t-rating-icon-filled"
+        [src]="iconFilled"
+        (click)="setRateByIndex(index)"
+    ></tui-svg>
+</div>
+
+<div class="t-rating-wrapper" [style.width.%]="percent">
+    <tui-svg
+        *tuiRepeatTimes="let index of max"
+        class="t-rating-icon-filled"
+        [src]="iconFilled"
+    ></tui-svg>
+</div>
+
+<tui-svg
+    *tuiRepeatTimes="let item of max"
+    class="t-rating-icon-normal"
+    [src]="iconNormal"
+></tui-svg>
+
+<input
+    #focusableElement
+    type="range"
+    step="1"
+    class="t-rating-input"
+    [max]="max"
```

Comment:
We need to make min="1", otherwise it is possible to set 0 rating with keyboard, but not with mouse.